tools/packaging/static-build/kernel/build.sh

```diff
@@ -24,50 +28,66 @@ container_image="${KERNEL_CONTAINER_BUILDER:-$(get_kernel_image_name)}"
 MEASURED_ROOTFS=${MEASURED_ROOTFS:-no}
 kernel_builder_args="-a ${ARCH} $*"
 
-if [ "${MEASURED_ROOTFS}" == "yes" ]; then
+if [[ "${MEASURED_ROOTFS}" == "yes" ]]; then
 	info "build initramfs for cc kernel"
 	"${initramfs_builder}"
 	# Turn on the flag to build the kernel with support to
 	# measured rootfs.
 	kernel_builder_args+=" -m"
 fi
 
-if [ "${CROSS_BUILD}" == "true" ]; then
+if [[ "${CROSS_BUILD}" == "true" ]]; then
```

Comment:
Do we need to handle if this hasn't been set?
```suggestion
if [[ "${CROSS_BUILD:-1}" == "true" ]]; then
```